examples/pytorch/rect/README.md

```diff
@@ -0,0 +1,41 @@
+# **DGL Implementation of RECT (TKDE20)**
+
+This DGL example implements the GNN model **RECT** (or more specifically its supervised part **RECT-L**) proposed in the paper [Network Embedding with Completely-imbalanced Labels](https://ieeexplore.ieee.org/document/8979355). The authors' original implementation can be found [here](https://github.com/zhengwang100/RECT).
+
+
+
+## **Dataset and experimental setting**
+
+Two DGL's build-in datasets (Cora and Citeseer) with their default train/val/test settings are used in this example. In addition, as this paper considers the zero-shot (i.e., completely-imbalanced) label setting, those "unseen" classes should be removed from the training set, as suggested in the paper. In this example, in each dataset, we simply remove the 2-3 classes (i.e., these 2-3 classes are unseen classes) from the labeled training set. Then, we obtain graph embedding results by different models. Finally, with the obtained embedding results and the original balanced labels, we train a logistic regression classifier to evaluate the model performance.
+
+
+
+## **Usage** 
+
+`python main.py --dataset cora --gpu 0 --model-opt RECT-L --removed-class 0 1 2` #reproducing the RECT-L on "cora" datasets in the zero-shot label setting using GPU
+
+`python main.py --dataset cora --gpu 0 --model-opt GCN --removed-class 0 1 2` #reproducing the GCN on "cora" datasets in the zero-shot label setting using GPU
+
+`python main.py --dataset cora --gpu 0 --model-opt NodeFeats --removed-class 0 1 2` # evaluating the original node features using GPU
+
+
+
+## **Performance**
+
+The performance results are are as follows:
+
+| **Datasets/Models** | **NodeFeats** | **GCN** | **RECT-L** |
+| :-----------------: | :-----------: | :-----: | :--------: |
+|      **Cora**       |     47.56     |  51.26  | **68.60**  |
+|    **Citeseer**     |     42.04     |  37.55  | **56.32**  |

```

Comment:
Which setting are we using to get the performance? Does it match with the paper performance?

Reply:
Thanks.

Firstly, as described in the Readme.md, we adopt the standard split of DGL datasets and adopt some (user-specified) classes as unseen. 

Secondly, in the original paper, it randomly selects some percents of labeled data and also randomly adopts some unseen classes.
Therefore, we do not get the same performance.